consul/aio.py

```diff
@@ -42,8 +43,10 @@ def get(self, callback, path, params=None):
         uri = self.uri(path, params)
         return self._request(callback, 'GET', uri)
 
-    def put(self, callback, path, params=None, data=''):
+    def put(self, callback, path, params=None, data='', json=None):
```

Comment:
I did not notice that so I was following what `requests` is doing. Looking at the benefits of using `JSON` parameter, in `requests` library, it will set an `application/json` header for you. But with the other async lib, I have not checked if they are doing the same.